config/migrations/20201224101445-publication-flow-ingetrokken.sparql

```diff
@@ -0,0 +1,27 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+PREFIX pubstatusid: <http://kanselarij.vo.data.gift/id/concept/publicatie-statussen/>
+PREFIX activity-type: <http://kanselarij.vo.data.gift/id/concept/activiteit-types/>
+PREFIX publish-type: <http://kanselarij.vo.data.gift/id/concept/publicatie-types/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX euvoc: <http://publications.europa.eu/ontology/euvoc#>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    pub:Publicatiestatus a ext:PublicClass .
```

Comment:
Deze moet niet, dit hoeft maar 1 keer te gebeuren en is al in vorige migration gebeurd